src/pages/security/_hero.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { graphql, useStaticQuery } from 'gatsby'
+import HeroComponent from './components/_hero_component'
+import { Show } from 'components/containers'
+
+const query = graphql`
+    query {
+        hero_background_desktop: file(
+            relativePath: { eq: "security/security-overlay-desktop.jpg" }
+        ) {
+            ...heroImage
+        }
+        hero_background_mobile: file(relativePath: { eq: "security/security-overlay-mobile.jpg" }) {
+            ...heroImage
+        }
+    }
+`
+
+const Hero = () => {
+    const data = useStaticQuery(query)
+
+    return (
+        <>
+            <Show.Desktop>
+                <HeroComponent
+                    background_data={data['hero_background_desktop']}
+                    background_height="70vh"
```

Comment:
you can remove the height, and use padding instead in the `Container`
```suggestion
                    
```

Reply:
thank u very much for your revision. but I don't know how can I use padding instead of height! when I use padding-bottom: 100% the ui is not correct!